docs/enterprise-guide/full-app-embedding.md

```diff
@@ -1,85 +1,98 @@
 
 ### What you'll be doing
+
 To get this going, you're going to need:
-* An Enterprise Edition instance of Metabase that contains the dashboards and charts that you'd like to embed.
-* A separate web application that you want to embed your dashboards and charts in.
+
+- An Enterprise Edition instance of Metabase that contains the dashboards and charts that you'd like to embed.
+- A separate web application that you want to embed your dashboards and charts in.
 
 ### Enabling embedding in Metabase
+
 First, let's enable embedding in your Metabase instance. Go to the Admin Panel, and under Settings, go to the “Embedding in other applications” tab. From there, click “Enable.”
 
 Once you do, you'll see a set of options:
 
-* **Embedding secret key:** You can ignore this setting, which is only for standalone chart or dashboard embeds.
+- **Embedding secret key:** You can ignore this setting, which is only for standalone chart or dashboard embeds.
 
-* **Embedding the entire Metabase app:** Here's where you'll enter the base URLs of the web applications that you want to allow to embed Metabase.
+- **Embedding the entire Metabase app:** Here's where you'll enter the base URLs of the web applications that you want to allow to embed Metabase.
   This value will be used to populate the `Content-Security-Policy` HTTP header's [`frame-ancestors` directive](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/frame-ancestors), and should follow the same format.
   For example, `https://*.metabase.com http://my-web-app.example.com:8080/`. Leaving this empty will default to a `frame-ancestors` value of `'none'`.
   If you're a fancy person, you can specify this URL in the environment variable `MB_EMBEDDING_APP_ORIGIN`.
 
 A note to IE11 users: only the first URL will be valid for IE11 due to limitations in the HTTP headers it supports. IE11 does not support the `Content-Security-Policy` header, but does support `X-Frame-Options`, which can only accept a single value.
 
 ### Setting things up in your web app
+
 To give you a picture of what you'll need to do in your app, we've created this [reference app](https://github.com/metabase/sso-examples/tree/master/app-embed-example). If you use React in your application, [this React component](https://github.com/metabase/sso-examples/blob/master/app-embed-example/src/MetabaseAppEmbed.js) may be helpful.
 
-The main elements you'll need to embed Metabase in your app are:
+**Note:** When using an SSO authentication endpoint on a different domain than Metabase, to avoid problems with `SameSite` cookies, you might need to set the environment variable `MB_SESSION_COOKIE_SAMESITE=None`. The available options are `Lax` (default), `None` or `Strict`.
```

Comment:
This should be moved to directly after the bullet point below that begins with "An SSO authentication endpoint…"